### PR TITLE
10.3.0 consolidated envs

### PR DESCRIPTION
**Pull request by jasperRob**
This introduces some changes and resolves issue with DEF prefix.

### CODE REVIEW

File: bootq/src/main/java/life/genny/bootq/models/BatchLoading.java [hunk 1]
```diff
@@ -341,6 +345,13 @@ public void persistDefBaseEntityAttributes(Map<String, Map<String, String>> proj
 
             String baseEntityCode = row.get("baseentitycode");
             String attributeCode = row.get("attributecode");
+
```
Comment (varunshastry) [inline bootq/src/main/java/life/genny/bootq/models/BatchLoading.java:348]: Commented lines to be removed

File: bootq/src/main/java/life/genny/bootq/models/BatchLoading.java [hunk 2]
```diff
@@ -573,13 +593,18 @@ public void persistQuestionQuestions(Map<String, Map<String, String>> project, S
...
+            String targetCode = row.get("targetcode");
+            log.debug("persisting: " + parentCode + ":" + targetCode);
+
             try {
 				questionUtils.saveQuestionQuestion(questionQuestion);
 			} catch (Exception e) {
-                Map<String, String> row = entry.getValue();
-                String parentCode = row.get("parentcode");
-                String targetCode = row.get("targetcode");
+                // Map<String, String> row = entry.getValue();
```
Comment (varunshastry) [inline bootq/src/main/java/life/genny/bootq/models/BatchLoading.java]: Commented lines to be removed

File: bootq/src/main/java/life/genny/bootq/models/reporting/LoadReport.java [hunk 3]
```diff
@@ -198,20 +198,22 @@ public void printLoadReport(boolean showStackTraces) throws java.io.IOException
...
-            e.printStackTrace();
-        }
-        try(BufferedWriter writer = new BufferedWriter(fw)) {
-            for(Object msg : linesToOutput) {
-                writer.append(msg.toString());
-            }
-        }
-        if(fw != null)
-            fw.close();
+        // FileWriter fw = null;
```
Comment (varunshastry) [inline bootq/src/main/java/life/genny/bootq/models/reporting/LoadReport.java]: Commented lines to be removed

**Review by varunshastry — CHANGES_REQUESTED** [review]

File: bootq/src/main/java/life/genny/bootq/models/BatchLoading.java (hunk 1 above)
Comment (jasperRob) [reply bootq/src/main/java/life/genny/bootq/models/BatchLoading.java:348]: This one is left on purpose

**Comment by Expressionless** [comment]
File writing comments I would request to stay in. These are going to be readded next Pull Request.

**Review by varunshastry — APPROVED** [review]

**Comment by Expressionless** [comment]
Potential issue on null check added in EntityAttributeUtils
In QwandaUtils better to return empty array for codes rather than null (in this case initialize as new ArrayList<>() instead of null), saves us a redundant check and becomes SONARlint compliant

**Pull request merged**